code/modules/zombie/organs.dm

```diff
@@ -53,6 +53,7 @@
 		owner << "<span class='narsiesmall'>You can feel your heart stopping, but something isn't right... \
 		life has not abandoned your broken form. You can only feel a deep and immutable hunger that \
 		not even death can stop, you will rise again!</span>"
+		owner << "<span class = 'userdanger'>You are an antagonist.</span>" // because apparently the concept that zombies might be bad guys is too hard to grasp
```

Comment:
I'm not sure this is clear enough for our players. Perhaps the centcomm announcement font.